[AILITOOLS-268] Fix amdsmi_get_gpu_vram_vendor deprecation notice (#7203)
ROCm/rocm-systems: projects/amdsmi/CHANGELOG.md

### Changed
- **Deprecated `amdsmi_get_gpu_vram_vendor()` in favor of `amdsmi_get_gpu_vram_info()`**.  

>   - The API is slated for removal in a future ROCm release. It now emits a `DeprecationWarning` from the Python interface and is implemented on top of `amdsmi_get_gpu_vram_info()`.

Reviewer: I think it would be a bit more clear to say something like:

```suggestion
  - `amdsmi_get_gpu_vram_vendor` is slated for removal in a future ROCm release. It now emits a `DeprecationWarning` from the Python interface and functions as a wrapper of `amdsmi_get_gpu_vram_info()`.
```
